apps/scan/backend/src/scanners/pdi/pdi_app_scan.test.ts

```diff
@@ -222,6 +229,9 @@ test('ballot needs review - return', async () => {
       mockScanner.setScannerStatus(mockStatus.idleScanningDisabled);
       clock.increment(delays.DELAY_SCANNER_STATUS_POLLING_INTERVAL);
       await expectStatus(apiClient, { state: 'no_paper' });
+
+      // Make sure the ballot was still recorded in the db for backup purposes
+      expect(Array.from(workspace.store.forEachSheet())).toHaveLength(1);
```

Comment:
Discovered this missing assertion that I didn't copy over from the Custom tests when writing the PDI tests